### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by allwishes**
remove comments and add second print

### CODE REVIEW

**Comment by allwishes** [comment]
Hello)

File: web3_lesson_2/main.py [hunk 1]
```diff
@@ -11,13 +11,7 @@
 client.w3.middleware_onion.inject(geth_poa_middleware, layer=0)
 
 block = client.w3.eth.get_block('latest')
-# print(client.w3.eth.max_priority_fee)
```
Comment (CrashNosok) [inline web3_lesson_2/main.py:14]: why did you remove it?

**Comment by CrashNosok** [comment]
5+

**Comment by CrashNosok** [comment]
5+2
